src/plugins/workspace/server/plugin.ts

```diff
@@ -3,30 +3,50 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { Observable } from 'rxjs';
+import { first } from 'rxjs/operators';
 import {
   PluginInitializerContext,
   CoreSetup,
   Plugin,
   Logger,
   CoreStart,
+  SharedGlobalConfig,
 } from '../../../core/server';
+import {
+  WORKSPACE_SAVED_OBJECTS_CLIENT_WRAPPER_ID,
+  WORKSPACE_CONFLICT_CONTROL_SAVED_OBJECTS_CLIENT_WRAPPER_ID,
+} from '../common/constants';
 import { IWorkspaceClientImpl } from './types';
 import { WorkspaceClient } from './workspace_client';
 import { registerRoutes } from './routes';
-import { WORKSPACE_CONFLICT_CONTROL_SAVED_OBJECTS_CLIENT_WRAPPER_ID } from '../common/constants';
+import { WorkspaceSavedObjectsClientWrapper } from './saved_objects';
 import { WorkspaceConflictSavedObjectsClientWrapper } from './saved_objects/saved_objects_wrapper_for_check_workspace_conflict';
+import {
+  SavedObjectsPermissionControl,
+  SavedObjectsPermissionControlContract,
+} from './permission_control/client';
 
 export class WorkspacePlugin implements Plugin<{}, {}> {
   private readonly logger: Logger;
   private client?: IWorkspaceClientImpl;
   private workspaceConflictControl?: WorkspaceConflictSavedObjectsClientWrapper;
+  private permissionControl?: SavedObjectsPermissionControlContract;
+  private readonly globalConfig$: Observable<SharedGlobalConfig>;
+  private workspaceSavedObjectsClientWrapper?: WorkspaceSavedObjectsClientWrapper;
 
   constructor(initializerContext: PluginInitializerContext) {
     this.logger = initializerContext.logger.get('plugins', 'workspace');
+    this.globalConfig$ = initializerContext.config.legacy.globalConfig$;
   }
 
   public async setup(core: CoreSetup) {
     this.logger.debug('Setting up Workspaces service');
+    const globalConfig = await this.globalConfig$.pipe(first()).toPromise();
+    const isPermissionControlEnabled =
+      globalConfig.savedObjects.permission.enabled === undefined
+        ? true
+        : globalConfig.savedObjects.permission.enabled;
```

Comment:
do we want to enable permission control on saved objects by default?

Reply:
The `permission.enabled` in saved objects was false by default. I think we don't need to do these. Do you mean change here to 
```ts
    const isPermissionControlEnabled = globalConfig.savedObjects.permission.enabled;
```
?

Reply:
if it has default value, we might don't need this check ` globalConfig.savedObjects.permission.enabled === undefined`